### PR TITLE
delete

### PR DESCRIPTION
**Pull request by matt-fidd**

### CODE REVIEW

File: .test [hunk 1]
```diff
@@ -1 +1,7 @@
-ob
```
Comment (LazyAssassin445) [inline .test:1]: why delete?

File: .test [hunk 2]
```diff
@@ -1 +1,7 @@
...
+no!
+
+this is good!
```
Comment (LazyAssassin445) [inline .test:3]: woo!!

File: .test [hunk 3]
```diff
@@ -1 +1,7 @@
...
+
+this is bad :(
```
Comment (LazyAssassin445) [inline .test:5]: bad :(

**Review by LazyAssassin445 — CHANGES_REQUESTED** [review]

File: .test (hunk 1 above)
Comment (matt-fidd) [reply .test:1]: changed

**Review by LazyAssassin445 — APPROVED** [review]

**Review by LazyAssassin445 — APPROVED** [review]

**Review by LazyAssassin445 — APPROVED** [review]

**Review by LazyAssassin445 — APPROVED** [review]

**Review by LazyAssassin445 — APPROVED** [review]

File: .test (hunk 2 above)
Comment (LazyAssassin445) [inline .test:3]: Change this

**Review by LazyAssassin445 — CHANGES_REQUESTED** [review]

File: .test (hunk 3 above)
Comment (LazyAssassin445) [inline .test:5]: change this too!

**Review by LazyAssassin445 — CHANGES_REQUESTED** [review]

File: .test [hunk 4]
```diff
@@ -1 +1,7 @@
-ob
+no!
```
Comment (LazyAssassin445) [inline .test:1]: this too

**Review by LazyAssassin445 — CHANGES_REQUESTED** [review]

**Review by LazyAssassin445 — CHANGES_REQUESTED** [review]
change!

**Review by LazyAssassin445 — APPROVED** [review]

**Review by LazyAssassin445 — APPROVED** [review]

**Review by LazyAssassin445 — APPROVED** [review]

**Review by LazyAssassin445 — APPROVED** [review]

**Review by LazyAssassin445 — APPROVED** [review]

**Pull request closed without merging**